### PR TITLE
Adding modules for planning Assembly (AAP-23356)

### PR DESCRIPTION
**Pull request by g-murray**
[AAP-23356 ](https://issues.redhat.com/browse/AAP-23356 )SaaS documentation - Add planning chapter

Cretaing and linking the modules for the Planning chapter assembly

Location : aap-clouds-latest > stories > topics

### CODE REVIEW

File: stories/topics/ref-saas-planning-configs.adoc [hunk 1]
```diff
@@ -0,0 +1,5 @@
+[id="ref-saas-planning-configs"]
+= Planning the required configurations for {SaaSonAWSShort}
+After accessing your {PlatformNameShort} instance, you can confirm that there is only a control node, and mesh ingress hop nodes configured on the *Instances* page or *Topology* page.
```
Comment (ianf77) [inline stories/topics/ref-saas-planning-configs.adoc:3]: You configure the nodes on the Create Instances page and view the results using the Topology viewer.

File: stories/topics/ref-saas-planning-configs.adoc [hunk 2]
```diff
@@ -0,0 +1,5 @@
+[id="ref-saas-planning-configs"]
+= Planning the required configurations for {SaaSonAWSShort}
+After accessing your {PlatformNameShort} instance, you can confirm that there is only a control node, and mesh ingress hop nodes configured on the *Instances* page or *Topology* page. 
+Before you can start automation jobs you must add remote execution nodes to the cluster, or connect an OpenShift cluster as a container group. 
+For help with configuring execution nodes see link:https://docs.redhat.com/en/documentation/red_hat_ansible_automation_platform/2.4/html-single/red_hat_ansible_automation_platform_automation_mesh_guide_for_vm-based_installations/index#assembly-planning-mesh[Red Hat Ansible Automation Platform Automation Mesh Guide for VM-based installations]
```
Comment (ianf77) [inline stories/topics/ref-saas-planning-configs.adoc:5]: If you're creating instances (e.g. execution nodes are you creating them entirely using inventory files (i.e a VM-based install) or are you creating them using Controller (an operator-based install), or both. I think you need to distinguish which one you're using.

**Review by ianf77 — COMMENTED** [review]
Some comments.

File: stories/topics/ref-saas-planning-configs.adoc (hunk 2 above)
Comment (g-murray) [reply stories/topics/ref-saas-planning-configs.adoc:5]: I have asked Scott after your comment from Friday. Just waiting on his response.

File: stories/topics/ref-saas-planning-configs.adoc (hunk 1 above)
Comment (g-murray) [reply stories/topics/ref-saas-planning-configs.adoc:3]: Hey Ian, appreciate  the callout here, but just confirming via SaaS the nodes are configured automatically (after the user submits the registration form on AWS), this step is just telling the user where to look to confirm for themselves, they won't actually need to configure these. 

Maybe if this sentence is confusing I could say something like:
"you can confirm that there is only a control node, and mesh ingress hop nodes on the *Instances* page or *Topology* page. "

File: stories/topics/ref-saas-planning-configs.adoc (hunk 2 above)
Comment (g-murray) [reply stories/topics/ref-saas-planning-configs.adoc]: Scott confirmed:
"The operator-based installation guide, not the VM-based, will be the appropriate link.

The whole guide will be the context...the customer will have to follow the entire process to setup the automation mesh."

File: stories/topics/ref-saas-planning-configs.adoc (hunk 1 above)
Comment (ianf77) [reply stories/topics/ref-saas-planning-configs.adoc]: That would work.

**Review by ianf77 — APPROVED** [review]
LGTM

**Pull request merged**